lib/utils/note-utils.js

```diff
@@ -17,12 +28,19 @@ const noteTitleAndPreviewRegExp = /(\S[^\n]*)\s*(\S[^\n]*)?/;
  */
 export const noteTitleAndPreview = note => {
   const content = note && note.data && note.data.content;
-  const match = noteTitleAndPreviewRegExp.exec(content || '');
+
+  const match = isMarkdown(note)
+    ? noteTitleAndPreviewMdRegExp.exec(content || '')
+    : noteTitleAndPreviewRegExp.exec(content || '');
 
   const title = (match && match[1] && match[1].slice(0, 200)) || 'New note...';
   const preview = (match && match[2]) || '';
 
   return { title, preview };
```

Comment:
so this would end up being "above and beyond" this PR, but maybe it would be a good time to add it in. how would you feel about making these result indices clearer by eliminating them?

```js
const match = …;

if ( ! match ) {
	return { title: 'New note…', preview: '' };
}

const [ /* full match */, title, preview ] = match;

return {
	title: title.slice( 0, 200 ) || 'New note…',
	preview: preview || '',
};
```

I can see value in doing this and also in leaving it the same way.

Reply:
Looks clearer. I'm on it

Reply:
Thanks! this shouldn't have been your burden but I noted it simply as a result of the fact that the indices ended up being important in this PR

Reply:
actually we might as well also skip the pattern matching when the content is empty. no need. I like the cleanliness of `( content || '' )` but actually that's also needless.

would you be interested in catching empty-content cases above the match and return the defaults before then?

```js
if ( ! content ) {
	return {
		title: 'New note…',
		preview: '',
	}
}
```